gamemodes/terrortown/gamemode/client/cl_voice.lua

```diff
@@ -70,15 +130,11 @@ function GM:PlayerStartVoice(ply)
 		if client:IsActive() and not clrd.unknownTeam and not clrd.disabledTeamVoice then
 			local tm = client:GetTeam()
 			if tm ~= TEAM_NONE and not TEAMS[tm].alone then
-				if not client:KeyDown(IN_SPEED) and not client:KeyDownLast(IN_SPEED) then
-					client[tm .. "_gvoice"] = true
-
-					RunConsoleCommand("tvog", "1")
-				else
-					client[tm .. "_gvoice"] = false
-
-					RunConsoleCommand("tvog", "0")
-				end
+				net.Start("TTT2RoleGlobalVoice")
+				local state = not VOICE.IsTeam
+				client[tm .. "_gvoice"] = state
```

Comment:
maybe rearrange that a bit. create the car first and change it, leave and an empty line prior to the net-stuff

Reply:
Done